spring-data-jpa/src/test/java/org/springframework/data/jpa/repository/sample/UserRepository.java

```diff
@@ -544,7 +544,7 @@ List<User> findUsersByFirstnameForSpELExpressionWithParameterIndexOnlyWithEntity
 
 	List<RolesAndFirstname> findRolesAndFirstnameBy();
 
-	@Query(value = "FROM User u")
+	@Query(value = "SELECT u FROM User u")
```

Comment:
I think we should not alter the query here since it's valid JPQL even without the `SELECT` just because the Eclipselink preview is unable to deal with it. 
```
Problem compiling [FROM User u]
[7, 11] The identification variable 'this' is not defined in the FROM clause. (SELECT  [ this ] ...
	at org.eclipse.persistence.internal.jpa.jpql.HermesParser.buildException(HermesParser.java:175)
```